tests/integration/cleanup_resources.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import argparse
+import json
+import os
+import subprocess
+
+
+def cleanup_chaos_mesh(namespace) -> None:
+    print(f"Cleaning up chaos mesh in {namespace}")
+    env = os.environ
+    env["KUBECONFIG"] = os.path.expanduser("~/.kube/config")
+
+    output = subprocess.check_output(
+        f"tests/integration/ha_tests/scripts/destroy_chaos_mesh.sh {namespace}",
+        shell=True,
+        env=env,
+    )
+    for line in output.decode("utf-8").splitlines():
+        print(line)
+
+
+def cleanup_juju_models() -> None:
+    for model in _filter_tests_models("admin/test-"):
+        print(f"Destroying model {model}")
+        delete_cmd = ["juju", "destroy-model", model, "--destroy-storage", "-y"]
+        subprocess.check_output(delete_cmd).decode("utf-8")
```

Comment:
I don't think you need to `decode("utf-8")` since you don't use the output

Reply:
Done, thank you